pyOlog/ophyd_tools.py

```diff
@@ -446,10 +446,12 @@ def _print_pos(positioners, file=sys.stdout):
             continue
         if v is not None:
             try:
-                prec = p.precision
-            except (AttributeError, DisconnectedError):
-                prec = FMT_PREC
-            value = np.round(v, decimals=prec)
+                value = np.round(v, decimals=prec)
```

Comment:
This is going to raise an NameError.  We need two try/excepts here, one to cover (maybe) getting the precision from the PV and one to cover (maybe) rounding it.

Reply:
```python
try:
    perc = p.precision
except (AttributeError, DisconnectedError):
    perc = FMT_PERC

try:
    value = np.round(v, decimals=prec)
except TypeError:
    value = v
```

Reply:
OK, I am not seeing how this will result in a different outcome. but I am sure it is my own ignorance !-) I will play around myself and see why it won't work, in the meantime I will make this change.